### PR TITLE
Adding some quotes

### PR DESCRIPTION
**Pull request by bendai94**
Adding some Willy Wonka Quotes to the EC responses for fun

### CODE REVIEW

**Review by AverageHelper — CHANGES_REQUESTED** [review]
Got a couple extra files in here. Is the .vs directory needed, or can it be added to .gitignore?

Also, mind incrementing the responses count at the bottom of the textResponses file?

File: src/constants/textResponses.ts [hunk 1]
```diff
@@ -122,6 +122,17 @@ export const phrases: ResponseRepository = [
 	"The river Temarc... in winter",
 	"You know, back when I was in the academy, we would follow every toast with a song!",
 
+	// Wonka
+	"We are the music makers and we are the dreamers of the dreams.",
```
Comment (AverageHelper) [inline src/constants/textResponses.ts:126]: Would you sort these quotes alphabetically?

**Review by AverageHelper — CHANGES_REQUESTED** [review]

File: src/constants/textResponses.ts (hunk 1 above)
Comment (bendai94) [reply src/constants/textResponses.ts:126]: good advice, will do

**Review by AverageHelper — APPROVED** [review]
Looks good to me! I'll try the integration tests, but there's a good chance I messed them up and I'll have to merge this into a side branch first.

**Pull request merged**